node/crypto.ts

```diff
@@ -157,5 +150,15 @@ export function getHashes(): readonly string[] {
   return digestAlgorithms;
 }
 
-export default { Hash, createHash, getHashes, pbkdf2, pbkdf2Sync, randomBytes };
-export { pbkdf2, pbkdf2Sync, randomBytes };
+const randomUUID = globalThis.crypto.randomUUID;
```

Comment:
```suggestion
const randomUUID = () => crypto.randomUUID();
```
This should fix the `Illegal invocation` error

Reply:
Ah perfect! Thanks Divy!